### DISCUSSION TITLE
Welcome to NASA-Openscapes.github.io Discussions!

### DISCUSSION THREAD

**Discussion by erinmr** [NASA-Openscapes/NASA-Openscapes.github.io#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jules32** [comment]
Hello! 

I'm Julie, co-lead of NASA-Openscapes with Erin Robinson. This is my first time using GitHub Discussions!
